quiche/src/lib.rs

```diff
@@ -8583,6 +8593,8 @@ impl<'a> Iterator for UnknownTransportParameterIterator<'a> {
 pub struct TransportParams {
     /// Value of Destination CID field from first Initial packet sent by client
     pub original_destination_connection_id: Option<ConnectionId<'static>>,
+    /// The initial RTT
+    pub initial_rtt: Duration,
```

Comment:
This isn't really a QUIC transport parameter, so I think it'd make more sense to add it directly to the `Config` struct.